src/dotnet/commands/dotnet-add/dotnet-add-proj/Program.cs

```diff
@@ -104,9 +105,93 @@ private void AddProject(SlnFile slnFile, string projectPath)
                 };
 
                 slnFile.Projects.Add(slnProject);
+
+                EnsureBuildConfigurationsAreValidAndComplete(slnFile);
+
                 Reporter.Output.WriteLine(
                     string.Format(CommonLocalizableStrings.ProjectAddedToTheSolution, projectPath));
             }
         }
+
+        private void EnsureBuildConfigurationsAreValidAndComplete(SlnFile slnFile)
+        {
+            HashSet<string> _buildConfigurations = new HashSet<string>()
+            {
+                "Debug",
+                "Release",
+            };
+
+            HashSet<string> _buildPlatforms = new HashSet<string>()
+            {
+                "Any CPU",
+                "x64",
```

Comment:
These are the ones required by the c#, f#, VB, net core templates to ensure that it produces the same sln as VS (or at least an sln file that VS accepts and won't try to change when you open it).